core/iwasm/interpreter/wasm_loader.c

```diff
@@ -4173,6 +4202,10 @@ load_export_section(const uint8 *buf, const uint8 *buf_end, WASMModule *module,
         }
     }
 
+    if (!check_duplicate_exports(module, error_buf, error_buf_size)) {
+        return false;
+    }
```

Comment:
had better put the three lines before L4203, we only need to check it when export count is larger than 0.

Reply:
And could you apply this optimization for AOT? See
https://github.com/bytecodealliance/wasm-micro-runtime/blob/main/core/iwasm/aot/aot_loader.c#L2778-L2784

Reply:
Fixed, also made the change for AOT